zyte_common_items/components.py

```diff
@@ -285,3 +285,42 @@ class RealEstateArea(Item):
 
     #: Area in the raw format, as it appears on the website.
     raw: str
+
+
+@attrs.define(kw_only=True)
+class Header(Item):
+    """A http header value"""
+
+    #: Name of header
+    name: str
+
+    #: Value of the header
+    value: str
+
+
+@attrs.define(kw_only=True)
+class Request(Item):
+    """Describe a HTTP request to load a page"""
+
+    #: Name of the page which describes the URL.
+    name: Optional[str] = None
+
+    #: HTTP URL
+    url: str = attrs.field(converter=url_to_str)
+
+    #: HTTP Method
+    #:
+    #: Values: "GET" "POST" "PUT" "DELETE" "OPTIONS" "TRACE" "PATCH" "HEAD" "CONNECT"
```

Comment:
```suggestion
    #: HTTP method
```

Since we are supporting all HTTP method values, I don’t think we need to name it. We may even support arbitrary ones, right?